src/main/java/de/adesso/anki/battle/renderers/MqqtRenderer.java

```diff
@@ -0,0 +1,52 @@
+package de.adesso.anki.battle.renderers;
+
+import de.adesso.anki.battle.mqtt.MqttService;
+import de.adesso.anki.battle.transfer.BodyDTO;
+import de.adesso.anki.battle.transfer.RoadmapDTO;
+import de.adesso.anki.battle.world.World;
+
+import org.eclipse.paho.client.mqttv3.MqttClient;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.eclipse.paho.client.mqttv3.MqttClient'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/adessoag-github/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.adesso.anki%3Aanki-battle&pullRequest=32&issues=AWSshIs_sk71k7lwCc1c&open=AWSshIs_sk71k7lwCc1c)

[ISSUE_KEY]: AWSshIs_sk71k7lwCc1c